graph/path/internal/testgraphs/limited.go

```diff
@@ -161,12 +161,13 @@ func (l *LimitedVisionGrid) NodeAt(r, c int) graph.Node {
 	return l.Grid.NodeAt(r, c)
 }
 
-// Has returns whether n is a node in the grid.
+// Has returns whether the node with the given ID is a node in the grid.
 func (l *LimitedVisionGrid) Has(id int64) bool {
 	return 0 <= id && id < int64(len(l.Grid.open))
 }
 
-// Node return a node from g if it exists.
+// Node returns the node in the graph with the given ID if it exists,
+// and nil otherwise.
```

Comment:
As `LimitedVisionGrid` does not actually store the node, should this be pointed out in the comment? The same goes for `Grid` (both of which return a new `simple.Node`).

Reply:
Whether the node is stored or not is not relevant here. But also, these are internal testing packages, so I'm not particularly fussed.